### ISSUE TITLE
Blossom Bounce is broken.

### ISSUE THREAD

**Issue by IdaDood** [Reeds-github/Hero-Mode#7]
I'm assuming Blossom Bounce is meant to be obtained much later on, but was added as a rare drop for one of the Boggly Woods enemies. I actually got it through the latter. (note im not fully through the chapter yet lmao)
Blossom Bounce is entirely broken, though, in more ways than one. The description says that it uses 3 HP to attack enemies and gain 1 FP per hit, along with a chance of lowering their damage output. The last part there works fine, but every other part is broken.

The badge does not use HP to use. Instead, it uses 2 FP. On top of that, the actual intended purpose of gaining FP from dealing hits is just straight up non-functional. I'm assuming you used the code for the enemy attacks that drain FP and not HP, because instead of giving you FP from attacks, it instead _takes your own._ You are literally using FP to lose even more FP. So that's very fun.

TLDR: Blossom Bounce is supposed to use 3 HP to regen 1 FP per hit. Instead, it uses 2 FP and also drains your own FP by 1 per hit.
